packages/core-data/src/private-selectors.ts

```diff
@@ -253,3 +253,7 @@ export const getTemplateId = createRegistrySelector(
 		} );
 	}
 );
+
+export function getTemplateAutoDraftId( state: State, target: string ) {
```

Comment:
What is "target" here? is it `templateSlug` or something ?

Reply:
Renamed it